README.adoc

```diff
@@ -83,9 +85,6 @@ Type: ClusterIP
 ...
 ----
 
-WARNING: The proxy will only serve requests without the text/html accept header,
-using the browser directly won't work.
```

Comment:
I'm not sure to understand. When I go to just `http://localhost:3000` from my browser, it uses `Accept: text/html` and the proxy works. So this is confusing (to me at least) because, how I understand that sentence, it shouldn't work.

If I run `curl localhost:3000/api` then I'm given the json response.
If I run `curl localhost:3000/api -H "Accept: text/html"` it returns the front page HTML.

Isn't it a normal / expected behaviour?

Reply:
Is normal, yes.
I added the warning (IIRC I took it from the react page) to prevent users from testing if the proxy was OK by pointing their browsers to http://localhost:3000/api - maybe is not needed so I'm ok with removing it.